test/go.mod

```diff
@@ -49,6 +49,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
+replace github.com/IBM/TSS/mpc/binance/ecdsa => ../mpc/binance/ecdsa
```

Comment:
no, let's not do this. 

You can make a change to the `mpc/binance/ecdsa` package in this PR, and then wait for this PR to be merged, and then make a new PR that would just take the latest version into the `go.mod` of the `test` module.